shared/common-adapters/input.native.js

```diff
@@ -208,9 +208,10 @@ class Input extends Component<Props, State> {
 
     const multilineStyle = {
       ...commonInputStyle,
-      paddingTop: 0,
-      paddingBottom: 0,
+      height: undefined,
```

Comment:
flow doesn't like us adding height dynamically later w/o it being defined first, that seems fine to me